cocos/physics/framework/components/collider/collider-component.ts

```diff
@@ -176,6 +176,39 @@ export class ColliderComponent extends Eventify(Component) {
 
     constructor () { super() }
 
+    /**
+     * @zh
+     * 注册触发事件或碰撞事件相关的回调。
+     * @param type - 触发或碰撞事件的类型，可为 'onTriggerEnter'，'onTriggerStay'，'onTriggerExit' 或 'onCollisionEnter'，'onCollisionStay'，'onCollisionExit';
+     * @param callback - 注册的回调函数
+     * @param target - 可选参数，执行回调函数的目标
```

Comment:
- API description should have English version
- Parameter API comment should all be in English